test/examples/test_kissgp_additive_regression.py

```diff
@@ -14,7 +14,7 @@
 from torch.autograd import Variable
 from gpytorch.kernels import RBFKernel, AdditiveGridInterpolationKernel
 from gpytorch.likelihoods import GaussianLikelihood
-from gpytorch.means import ConstantMean
+from gpytorch.means import ZeroMean
```

Comment:
should we run the test against both ZeroMean and ConstantMean?

Reply:
All of our other tests use ConstantMean - I just wanted there to be one test with ZeroMean.